run/GCClassic/HISTORY.rc.templates/HISTORY.rc.CH4

```diff
@@ -101,7 +101,7 @@ COLLECTIONS: 'Restart',
   SpeciesConc.duration:       ${RUNDIR_HIST_TIME_AVG_DUR}
   SpeciesConc.mode:           'time-averaged'
   SpeciesConc.fields:         'SpeciesConcVV_?ALL?           ',
-                              ##'SpeciesConcMND_?ALL?          ',
```

Comment:
Why is this change made here and in other HISTORY.rc files for both GC-Classic and GCHP? @1Dandan, could you add a description of the diagnostic default changes to the main description of the PR and why you made them?